.circleci/config.yml

```diff
@@ -177,6 +177,7 @@ executors:
       CUSTOMS_SERVER_URL: none
       HUSKY_SKIP_INSTALL: 1
       AUTH_CLOUDTASKS_USE_LOCAL_EMULATOR: true
+      RATE_LIMIT__RULES: ""
```

Comment:
We are doing this in the CI so that rate limiting doesn't interfere with our playwright tests. Customs wasn't in play for playwright tests before (see `CUSTOMS_SERVER_URL:none` a couple lines up), so this feels symmetrical. It's worth noting that FXA-11663, will provide options to ignore certain IPs, emails, or UIDs. In the event that we do want to have customs invoked in a functional tests, which might be a nice test suite to have given some of the gaps we've seen with customs related issues. I'm thinking down the road we could use these filters to at least exercise some of the customs specific workflows with playwright tests.